browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -4636,6 +4661,61 @@ index f423a4a1d5399326fc48fe4d4a8a8fb9d4df861e..b4b60162d8b0d34113df052b04a1695d
          static String singleCharacterString(unsigned);
  #endif
  
+diff --git a/Source/WebCore/platform/PlatformScreen.cpp b/Source/WebCore/platform/PlatformScreen.cpp
+index ba50b688ab6d0bae5d199fa0bac4b7e2004baf81..0eb3267c45dfae821aa96435e3f95621d8c0ea98 100644
+--- a/Source/WebCore/platform/PlatformScreen.cpp
++++ b/Source/WebCore/platform/PlatformScreen.cpp
+@@ -25,6 +25,7 @@
+ 
+ #include "config.h"
+ #include "PlatformScreen.h"
++#include "RuntimeEnabledFeatures.h"
+ 
+ #if PLATFORM(COCOA)
+ 
+@@ -72,3 +73,20 @@ const ScreenData* screenData(PlatformDisplayID screenDisplayID)
+ } // namespace WebCore
+ 
+ #endif // PLATFORM(COCOA)
++
++#if ENABLE(TOUCH_EVENTS)
++namespace WebCore {
++
++bool screenHasTouchDevice() {
++    if (RuntimeEnabledFeatures::sharedFeatures().touchEventsEnabledOverridden())
```

Comment:
Why not move this logic to RuntimeEnabledFeatures instead?

Reply:
I assume you mean moving consumers of this api to call directly into RuntimeEnabledFeatures? I would need to add another feature for PrimaryInputDevice to make that work. The diff was much larger, so I decided to go with this.

Reply:
I think i figured out what you meant, ptal